### ISSUE TITLE
0.1.7 "Orphaned templates from tracker" does not work in issue templates list page of project.

### ISSUE THREAD

**Issue by kazztomo** [akiko-pusu/redmine_issue_templates#150]
Hi akiko, thank you for providing this plugin.
I have found an issue below:

When I click 'Orphaned templates from tracker' link in Project Settings issue template list page, there is no list is shown.
And this error is logged.

```
Started GET "/redmine/projects/issue_templates/issue_templates/orphaned_templates" for 192.168.1.30 at 2017-08-04 11:29:51 +0900
Processing by IssueTemplatesController#orphaned_templates as JS
  Parameters: {"project_id"=>"issue_templates"}
  Current user: admin (id=1)
Completed 500 Internal Server Error in 12ms (ActiveRecord: 0.8ms)

NameError (undefined local variable or method `class_name' for #<Class:0x007f5c3fe20ed0>
Did you mean?  class_eval):
  plugins/redmine_issue_templates/app/models/concerns/issue_template/common.rb:33:in `block (2 levels) in <module:Common>'
  plugins/redmine_issue_templates/app/controllers/issue_templates_controller.rb:152:in `orphaned_templates'
  lib/redmine/sudo_mode.rb:63:in `sudo_mode'
```

my environment:

```
Redmine  3.4.2.stable
Ruby     2.4.1-p111 (2017-03-22) [x86_64-linux]
Rails    4.2.8
```

If I change `class_name` to `self.name` in 'plugins/redmine_issue_tempaltes/app/models/concerns/issue_template/commn.rb:33', it looks like be worked right.

Thanks.

**Comment by akiko-pusu** [comment]
Hi, @kazztomo
Thanks for your repporting issue and helpful information.
I could not reproduce this situation yet, but 

Related line is here:

https://github.com/akiko-pusu/redmine_issue_templates/blob/master/app/models/concerns/issue_template/common.rb#L33

**Comment by akiko-pusu** [comment]
I’ve just confirmed and reproduced above situation, thank you so much!
So, I’m going to fix and release as soon as possible.

**Comment by kazztomo** [comment]
Thank you!
I'm looking forward to be released it.

**Comment by akiko-pusu** [comment]
NOTE: In suppor-rails5 branch, this was already corrected.
https://github.com/akiko-pusu/redmine_issue_templates/pull/147/commits/0ad6e3edf1deaa499a145da880e9befe3e404fc2#diff-30a41d466672c9e23e8d32c8c7c18214R34

**Comment by akiko-pusu** [comment]
Hi, @kazztomo 
I’ve just fixed and released v0.1.8. Thank you so much for your supporting!

**Comment by kazztomo** [comment]
Hi, @akiko-pusu 
Thank you for fixing and releasing.
I'll try to test and report on the next week.
